packages/relay/src/lib/eth.ts

```diff
@@ -573,8 +574,9 @@ export class EthImpl implements Eth {
         },
         requestIdPrefix,
       );
-      if (contractCallResponse?.result) {
-        gas = prepend0x(contractCallResponse.result);
+
+      if (contractCallResponse && contractCallResponse.result) {
```

Comment:
Out of curiosity, how does the old condition differ from this one?

Reply:
Ah yeah so the "?." operator allows for accessing properties of an object that maybe null or undefined without causing error. On the other hand, the `&&` condition firstly checks if contractCallResponse exists or not, and then access the result property. If contractCallResponse is null or undefined it will not attempt to access to the result property. Huh now that it's broken down it seems like they both eventually serve the same purpose in this context.